README.md

```diff
@@ -1,33 +1,38 @@
 # <img src="docs/_static/afkak.png" width="23" height="36" alt=""> Afkak: Twisted Python Kafka Client
 
-This module provides low-level protocol support for [Apache Kafka][kafka] as well as
-high-level consumer and producer classes. Request batching is supported by the
-protocol as well as broker-aware request routing. Gzip and Snappy compression
-is also supported for message sets.
+<a href="https://pypi.org/projects/afkak"><img src="https://img.shields.io/pypi/v/afkak.svg" alt="PyPI"></a>
+<a href="./LICENSE"><img src="https://img.shields.io/pypi/l/afkak.svg" alt="Apache 2.0"></a>
+<a href="https://afkak.readthedocs.io/en/latest/"><img src="https://readthedocs.org/projects/pip/badge/" alt="Documentation"></a>
+<!--
+TODO: Uncomment this once the build is less flaky.
+<a href="https://travis-ci.org/ciena/afkak"><img src="https://travis-ci.org/ciena/afkak.svg?branch=master" alt="Travis CI"></a>
+-->
 
-[kafka]: http://kafka.apache.org/
+<!--
+Everything between the LONG_DESCRIPTION_START and LONG_DESCRIPTION_END
+comments is taken as the package long_description by setup.py. Do not
+change the formatting of these lines lest that break.
+-->
+<!-- LONG_DESCRIPTION_START -->
 
-# License
-
-Copyright 2013, 2014, 2015 David Arthur under Apache License, v2.0. See `LICENSE`
-
-Copyright 2014, 2015 Cyan, Inc. under Apache License, v2.0. See `LICENSE`
-
-Copyright 2015, 2016, 2017, 2018 Ciena Corporation under Apache License, v2.0. See `LICENSE`
+Afkak is a [Twisted](https://twistedmatrix.com/)-native [Apache Kafka][https://kafka.apache.org/] client library.
```

Comment:
The https://kafka.apache.org/ link should to be surrounded by parenthesis instead of square brackets.